cranelift-wasm/Cargo.toml

```diff
@@ -11,7 +11,7 @@ keywords = ["webassembly", "wasm"]
 edition = "2018"
 
 [dependencies]
-wasmparser = { version = "0.49.0", default-features = false }
+wasmparser = { version = "0.51.0", default-features = false }
```

Comment:
wasmparser is at [0.51.2](https://crates.io/crates/wasmparser); do you want to bump to that?

Reply:
0.51.2 is semver-compatible, and we don't require anything from 0.51.2, so it doesn't matter either way.